src/components/Register/Registration.tsx

```diff
@@ -28,16 +26,20 @@ function Copyright(props: any) {
 const theme = createTheme()
 
 export const Registration: React.FC = () => {
-    const history = useHistory()
-    const isRegister = useSelector<RootState, RegistrationSuccessType>(state => state.registration.isRegistrationSuccess)
 
+    const isRegister = useTypedSelector(state => state.registerReducer.isRegistrationSuccess)
+    const [userRegStatus, setUserRegStatus] = useState("")
 
-    const historyHandler = () =>{
-        history.push("/login")
-    }
-    if (isRegister === "success") {
-        return <Redirect to={"/login"}/>
-    }
+    const userRegisteredStatus = useMemo(() => {
+        if (isRegister === "false") {
+            setUserRegStatus("User already registered, please enter new data")
+            return <div style={{color: "red"}}>{userRegStatus}</div>
```

Comment:
please move inline styles outside of the component, consider using css/scss modules and/or styled components